server/views/challenges/showBonfire.jade

```diff
@@ -30,15 +30,15 @@ block content
                                         ul: li: a(href=""+link, target="_blank") !{MDNkeys[index]}
                             .button-spacer
                 if (user)
-                label.btn.btn-primary.btn-block.btn-lg#submitButton
+                button.btn.btn-primary.btn-block.btn-lg#submitButton
                         | Run tests (ctrl + enter)
                 .button-spacer
                 .btn-group.input-group.btn-group-justified
-                    label.btn.btn-primary.btn-primary-ghost.btn-lg#trigger-reset-modal
+                    a.btn.btn-primary.btn-primary-ghost.btn-lg#trigger-reset-modal(href="#" role="button")
```

Comment:
Though `a` is serving the purpose to some extent, it's not a `button`, can't be triggered with `space`.
It should be `button`, to address the `display:table-cell`, wrap them in a `.btn-group`

Refer [bootstarp documentation](http://getbootstrap.com/components/#with-button-elements) for details.